common.js

```diff
@@ -1943,6 +1943,64 @@ const allSoloCars = {
       '2012': ['gs']
     },
   },
+  'Opel': {
+      '1900': ['fsp'],
```

Comment:
This structure does not match the rest of object. The structure in the rest of the object is
```
make: {
  model: { 
    year/all: [class1, class2]
  }
}
```

The structure you provided here is
```
make: {
  model: [class1, class2]
}
```

Please adjust this entry. Thanks!

Reply:
Ok i'll fix that